.eslintignore

```diff
@@ -0,0 +1,5 @@
+content
+docs
+dist
+scripts
+node_modules
```

Comment:
I usually just use a symlink to gitignore for this. (or use --exclude .gitignore in the eslint script if windows compat is needed).
But that might works too.